newlib/libm/test/math.c

```diff
@@ -676,8 +678,12 @@ test_math (int vector)
   test_asinh(vector);
   test_asinhf(vector);
   test_atan(vector);
+#endif
+#if TEST_PART > 0
```

Comment:
I think this is missing `|| TEST_PART == -1`.

Reply:
oops, missed this -- thanks.